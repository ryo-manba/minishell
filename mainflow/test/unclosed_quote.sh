echo "
echo ""
echo """
echo '
echo ''
echo '''
export VAR=var
echo "$VAR
echo $VAR"
echo '$VAR
echo $VAR'
echo "$VAR'
echo '$VAR"
