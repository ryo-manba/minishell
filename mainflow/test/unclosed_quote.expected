





var
var
$VAR
var
var'
$VAR"
